docs/multi_process.md

```diff
@@ -128,8 +131,8 @@ run a parallel computation across
 multiple processes. (If you’re not already familiar with how to use
 `shard_map` to run across multiple devices within a single process, check
 out the {doc}`/sharded-computation` tutorial.) Each process should call the
-same pmapped function and pass in arguments to be mapped across its *local*
-devices (i.e., the pmapped axis size is equal to the number of local devices).
+same jit function and pass in arguments to be mapped across its *local*
```

Comment:
Let's delete this entire sentence? It makes no sense to me to talk about jit and local devices